babel.config.js

```diff
@@ -121,6 +121,19 @@ module.exports = (api) => {
           ],
         ],
       },
+      {
+        test: 'packages/instantsearch-ui-components',
+        plugins: [
+          [
+            '@babel/plugin-transform-runtime',
+            {
+              corejs: false,
+              helpers: true,
+              regenerator: true,
```

Comment:
I think we want false, no? We don’t use async/await and we don’t want regenerator in the bundle.

Reply:
This is correct, it was in `@algolia/ui-components` which I copy/pasted as-is, but it isn't necessary. Removed in [9054a2b](https://github.com/algolia/instantsearch/pull/6037/commits/9054a2ba53a47b2b3ff29b70b647b175fbd242d1).